runelite-client/src/main/java/net/runelite/client/plugins/StatusBars/StatusBarsOverlay.java

```diff
@@ -0,0 +1,324 @@
+		Widget bottomLine = client.getWidget(WidgetInfo.RESIZABLE_VIEWPORT_BOTTOM_LINE_INTERFACE_CONTAINER);
+		if (widgetBankTitleBar != null)
+		{
+			return null;
+		}
+
+		if (client.isResized() && client.getWidget(WidgetInfo.RESIZABLE_VIEWPORT_BOTTOM_LINE_INTERFACE_CONTAINER) != null)
+		{
+			if (bottomLine.isHidden() && client.getWidget(WidgetInfo.RESIZABLE_VIEWPORT_OLD_SCHOOL_BOX).isHidden())
+			{
+				System.out.println("1");
+				return null;
+			}
+		}
+
+		// Get Max HP levels and current HP levels
+		final int maxHealth = client.getRealSkillLevel(Skill.HITPOINTS);
+		final int maxPrayer = client.getRealSkillLevel(Skill.PRAYER);
+		final int currentHealth = client.getBoostedSkillLevel(Skill.HITPOINTS);
+		final int currentPrayer = client.getBoostedSkillLevel(Skill.PRAYER);
+
+		// Check if player = poisoned or venomed and change color.
+		Color healthBar;
+		final int poisonState = client.getVar(VarPlayer.IS_POISONED);
+		if (poisonState > 0 && poisonState < 50)
+		{
+			healthBar = POISONED_COLOR;
+		}
+		else if (poisonState >= 1000000)
+		{
+			healthBar = VENOMED_COLOR;
+		}
+		else
+		{
+			healthBar = HEALTH_COLOR;
+		}
+
+		// Checks if the player has activated quick prayers
+		final int quickPrayerState = client.getVar(Varbits.QUICK_PRAYER);
+		final Color prayerBar = quickPrayerState == 1 ? QUICK_PRAYER_COLOR : PRAYER_COLOR;
+
+		// Render the health and prayer bars
+		renderBar(g, offsetHealthX , offsetHealthY,
+			maxHealth, currentHealth, height, healthBar);
+
+		renderBar(g, offsetPrayerX, offsetPrayerY ,
+			maxPrayer, currentPrayer, height, prayerBar);
+
+		// If the player hovers over a food or prayer restoring item this
+		// for loop will get the values from the item stats plugin
+		int prayerHealValue = 0;
+		int foodHealValue = 0;
+		WidgetItem[] inventory = queryRunner.runQuery(new InventoryWidgetItemQuery());
+		java.awt.Point mousePos = new java.awt.Point(client.getMouseCanvasPosition().getX(), client.getMouseCanvasPosition().getY());
+		for (WidgetItem item : inventory)
+		{
+			if (item.getCanvasBounds().contains(mousePos))
+			{
+				Effect change = itemStatChanges.get(item.getId());
+				if (change != null)
+				{
+					StatsChanges statsChanges = change.calculate(client);
+					for (StatChange c : statsChanges.getStatChanges())
+					{
+						if (c.getStat().getName() == "Hitpoints")
+						{
+							foodHealValue = Integer.parseInt(c.getTheoretical());
+						}
+
+						if (c.getStat().getName() == "Prayer")
+						{
+							prayerHealValue = Integer.parseInt(c.getTheoretical());
+						}
+					}
+				}
+			}
+		}
+
+		// Store prayer and health icon images.
+		Image healthImage = skillIconManager.getSkillImage(Skill.HITPOINTS, true);
+		Image prayerImage = skillIconManager.getSkillImage(Skill.PRAYER, false);
+
+		// If the prayer has skill icons disabled then assign noImage
+		if (!config.enableSkillIcon())
+		{
+			healthImage = NO_IMAGE;
```

Comment:
Just dont render the images when the skill icons are disabled instead of rendering empty images